AuthServ/AuthServer.cpp

```diff
@@ -93,10 +93,34 @@ void auth_init(AuthServer_Private& client)
     /* BCast Shard Capabilities */
     START_REPLY(e_AuthToCli_ServerCaps);
     uint32_t bufSzPos = client.m_buffer.tell();
+#ifdef DS_OU_COMPATIBLE
+    // OpenUru parses this as the start of a FileDownloadChunk message,
+    // which will be ignored,
+    // because no file download transaction is active.
+    client.m_buffer.write<uint32_t>(37);
+
+    // H'uru sees: 0x25 bytes of data, bit vector is 1 dword long, value is [caps]
+    // OpenUru sees: message type 0x25 (FileDownloadChunk), transaction ID 0x10000, error code 0x[caps]0000, file size 0x0000[caps] (continued below)
+    caps.write(&client.m_buffer);
+#ifdef DEBUG
+    uint32_t newBufPos = client.m_buffer.tell();
+    if ((newBufPos - bufSzPos) > (3 * sizeof(uint32_t)))
```

Comment:
This needs to be an equality check - if the `hsBitVector` data is shorter for some reason, the hardcoded size will be mismatched as well.

```suggestion
    if (newBufPos - bufSzPos != 3 * sizeof(uint32_t))
```